packages/contracts-bedrock/contracts/L1/ResourceMetering.sol

```diff
@@ -104,20 +98,14 @@ abstract contract ResourceMetering is Initializable {
                 // Update the base fee by repeatedly applying the exponent 1-(1/change_denominator)
                 // blockDiff - 1 times. Simulates multiple empty blocks. Clamp the resulting value
                 // between min and max.
-                newBaseFee = SignedMath.min(
-                    SignedMath.max(
-                        int256(
-                            (newBaseFee *
-                                (
-                                    FixedPointMathLib.powWad(
-                                        1e18 - (1e18 / BASE_FEE_MAX_CHANGE_DENOMINATOR),
-                                        int256((blockDiff - 1) * 1e18)
-                                    )
-                                )) / 1e18
-                        ),
-                        int256(MINIMUM_BASE_FEE)
-                    ),
-                    int256(uint256(type(uint128).max))
+                newBaseFee = _clampBaseFee(
+                    (newBaseFee *
+                        (
+                            FixedPointMathLib.powWad(
+                                1e18 - (1e18 / BASE_FEE_MAX_CHANGE_DENOMINATOR),
+                                int256((blockDiff - 1) * 1e18)
+                            )
+                        )) / 1e18
```

Comment:
I'm pretty sure we can do the clamping just once below here: 

```
    params.prevBaseFee = uint128(uint256(_clampBaseFee(newBaseFee)));
```

Reply:
Then the above can be rewritten a bit more cleanly as:
```
    newBaseFee *= (
        FixedPointMathLib.powWad(
            1e18 - (1e18 / BASE_FEE_MAX_CHANGE_DENOMINATOR),
            int256((blockDiff - 1) * 1e18)
        )
    );  
    newBaseFee /= 1e18;
```
(IDK why, but tests fail if I don't break the division out into another statement);